docs/documentation/telemetry.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: documentation
+title: Telemetry
+doc_order: 100
+description: About the use of Telemetry in mozregression
+---
+
+# Telemetry
+
+Unless explicitly disabled, mozregression will send an anonymized telemetry ping (a small JSON-document) to Mozilla's servers on each use (command-line invocation in the case of console mozregression, each bisection or single run in the case of the GUI).
+
+This data is anonymized and not personally identifiable, it contains only:
```

Comment:
I would be very cautious in stating this: while this ping doesn't contain PII, there's no active effort into performing deep anonymization.

Reply:
Good point, i'll remove the term "anonymization"